op-batcher/batcher/config.go

```diff
@@ -47,6 +47,13 @@ type CLIConfig struct {
 	// RollupRpc is the HTTP provider URL for the L2 rollup node.
 	RollupRpc string
 
+	// MaxChannelDuration is the maximum duration (in #L1-blocks) to keep the
+	// channel open. This allows control over how long a channel is kept open
+	// during times of low transaction volume.
```

Comment:
I think this is the best place to explain the behavior that there will be a first frame of a channel every `MaxChannelDuration + TxConfDepth` L1 blocks